Functions/Coverage.ps1

```diff
@@ -3,7 +3,8 @@ if ($PSVersionTable.PSVersion.Major -le 2)
     function Exit-CoverageAnalysis { }
     function Get-CoverageReport { }
     function Write-CoverageReport { }
-    function Enter-CoverageAnalysis {
+    function Enter-CoverageAnalysis
```

Comment:
I guess you used some automatic formatting but it created a lot of changes and introduced discrepancies with the rest of the source code. Could you undo that?